### PR TITLE
Update people section copy (Fixes #351)

### LINKED ISSUE #351: careers.mozilla.org - 'people page' copy revisions to 'How we take care of our people' section
| Estimates |  |
| ------------- | ------------- | 
| TL | |
| PM | 1 |
| ENG | |
| SEO | |
| ANA | |
| DES | |
| COPY | |

Description: The copy in the 'How we take care of our people' section on careers.mozilla.org is out of date and needs to be align with the current verbiage in use.
---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] The page is updated with new copy (and all the old, incorrect copy removed)
- [ ] The pdf describing benefits, comp, etc. is uploaded to assets.mozilla.org
- [ ] The pdf is link to in the page copy as in the design
- [ ] Clicks on the pdf are tracked in GA

### PR DESCRIPTION
**Pull request by alexgibson**
Fixes #351

- Removes perks section / carousel.
- Replaces with updated copy & link to PDF brochure.
- Tracks click event in GA.

### CODE REVIEW

File: careers/base/static/js/ga_event-tracking.js [hunk 1]
```diff
@@ -93,6 +93,11 @@
     });
 
 
+    /* Careers > People
+    ***************************************************************************/
+    trackClick('.life-people-cta', ['How We Take Care of Our People', 'link click', 'Learn more']);
```
Comment (stephaniehobson) [inline careers/base/static/js/ga_event-tracking.js]: I think it should be `How We Take Care of Our People Interactions` based on the other links being tracked.

File: careers/careers/static/js/careers.js [hunk 2]
```diff
@@ -4,7 +4,6 @@
     function animationInit() {
         teamsInit();
         galleryInit();
-        perksInit();
```
Comment (stephaniehobson) [inline careers/careers/static/js/careers.js]: Yay for code cleanup 🎉

File: careers/careers/templates/careers/home.jinja [hunk 3]
```diff
@@ -178,47 +178,13 @@
...
-            Are you a latte lover? Do you crave cookies? Or do you prefer to fuel up on fruit? Whatever you need, we have it. Or we can get it.
-          </p>
-        </div>
+    <article id="life-people">
+      <h3 class="life-people-head">How we take care of our people</h3>
+      <div class="life-people-desc">
+        <p>
+          We offer a unique, flexible work environment where individual contribution and impact produce results greater than a paycheck and benefits. But pay and benefits are important, so we offer market-competitive salaries and cash-based incentives as well as a full stack of benefits.
+        </p>
+        <a class="life-people-cta" href="https://assets.mozilla.net/pdf/mozilla-careers-comp-brochure.pdf">Learn more</a>
```
Comment (stephaniehobson) [inline careers/careers/templates/careers/home.jinja]: Asset is in place 👍

**Comment by alexgibson** [comment]
Thanks @stephaniehobson - I pushed another commit that addresses some small bits of stakeholder feedback (centering copy & using the star graphic) together with your GA suggestion.

**Comment by glogiotatidis** [comment]
Thanks!

**Pull request merged**